pkg/engine/virtualmachinescalesets.go

```diff
@@ -375,6 +377,8 @@ func CreateAgentVMSS(cs *api.ContainerService, profile *api.AgentPoolProfile) Vi
 		Tags: tags,
 	}
 
+	addCustomTagsToVMScaleSets(cs.Properties.MasterProfile.CustomVMTags, &virtualMachineScaleSet)
```

Comment:
same here `addCustomTagsToVM(profile.CustomVMTags, &virtualMachine)`

Reply:
Yeah, makes total sense! I got this in the wrong way and I just fixed this, lets see how the unit tests goes.